File: SearchAndFind-Frontend/app/src/main/java/uy/com/searchandfind/saler/SalerCategoryDTO.java
package uy.com.searchandfind.saler;

import uy.com.searchandfind.category.CategoryDTO;

/**
 * Created by Fede2 on 14/10/2017.
 */

public class SalerCategoryDTO {

    private boolean HasCategory;
    private CategoryDTO Category;
    private boolean IsUpdated;

    public boolean getHasCategory() {
        return HasCategory;
    }

    public void setHasCategory(boolean hasCategory) {
        this.HasCategory = hasCategory;
    }

    public CategoryDTO getCategory() {
        return Category;
    }

    public void setHasCategory(CategoryDTO category) {
        this.Category = category;
    }

    public boolean getIsUpdated() {
        return IsUpdated;
    }

    public void setIsUpdated(boolean isUpdated) {
        this.IsUpdated = isUpdated;
    }
}

File: SearchAndFindBackend/SearchAndFind.DTO/Saler/SalerCategoryDTO.cs
using SearchAndFind.Entities;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SearchAndFind.DTO
{
    public class SalerCategoryDTO 
    {
        public bool HasCategory { get; set; }
        public CategoryDTO Category { get; set; }
        public bool IsUpdated { get; set; }
    }
}
